content/content.go

```diff
@@ -33,6 +34,17 @@ import (
 const (
 	separator          = "------------------------------------------------------------"
 	directoryAttribute = "directory"
+
+	// PluginYAML represents the filename of rule's plugin specification
+	PluginYAML = "plugin.yaml"
+	// MetadataYAML represents the filename of rule error key's metadata
+	MetadataYAML = "metadata.yaml"
+
+	GenericMarkdown    = "generic.md"
```

Comment:
would it be possible please to put some comment about these file names?

Reply:
done! thank you